terraform/cloudwatch.tf.json.template.py

```diff
@@ -16,7 +16,8 @@
                     "period": "3600",
                     "statistic": "Average",
                     "threshold": "85",
-                    "alarm_description": "This metric monitors ES CPU utilization",
+                    "alarm_description": f'{{ "slack_channel": "dcp-ops-alerts",'
```

Comment:
We typically use `json.dumps()` if Terraform wants serialized JSON. Look for other uses of `json.dumps()` in other `*.tf.json.template.py` files for an example.